src/fxcorrelator.py

```diff
@@ -62,6 +64,7 @@ def __init__(self, descriptor, identifier=-1, config_source=None,
 
         """
         self.logger = logger
+        self.loghandler = None
```

Comment:
PyCharm says I should not assign class attributes outside of init. What do you think?